packages/catalog-process/src/services/catalogService.ts

```diff
@@ -2237,25 +2313,506 @@ export function catalogServiceBuilder(
 
       return updatedEService;
     },
-  };
-}
+    async updateTemplateInstanceName(
```

Comment:
Usually, for internal routes, we also name the methods `internal*` (see other services).
Samel for other internal calls in this file

```suggestion
    async internalUpdateTemplateInstanceName(
```

Reply:
Tracked here https://pagopa.atlassian.net/browse/PIN-6323